src/utils/getProgrammingLanguageName.ts

```diff
@@ -0,0 +1,23 @@
+import { CLASSNAME_PREFIX } from '../constants';
+
+const DEFAULT = 'bash';
+const TERMINAL = 'terminal';
+const JS = ['javascript', 'js', 'jsx', 'ts', 'tsx'];
+const SH = ['sh', 'shell'];
+const PY = ['python', 'py'];
+const SOL = ['solidity', 'sol'];
+const LANGS = [JS, SH, PY, SOL];
+
+export const getProgrammingLanguageName = (code: string) => {
+  for (const lang of LANGS) {
```

Comment:
Ah.. the second instance of it should be using `newCode` instead of the original form... pushed a commit to update this and added comments so it hopefully makes a bit more sense what is going on there.